qiita_db/test/test_artifact.py

```diff
@@ -327,6 +327,16 @@ def test_can_be_submitted_to_ebi(self):
         self.assertTrue(qdb.artifact.Artifact(3).can_be_submitted_to_ebi)
         self.assertFalse(qdb.artifact.Artifact(4).can_be_submitted_to_ebi)
 
+    def test_is_submitted_to_ebi(self):
+        with self.assertRaises(
+                qdb.exceptions.QiitaDBOperationNotPermittedError):
+            qdb.artifact.Artifact(1).is_submitted_to_ebi
+        self.assertTrue(qdb.artifact.Artifact(2).is_submitted_to_ebi)
+        self.assertFalse(qdb.artifact.Artifact(3).is_submitted_to_ebi)
+        with self.assertRaises(
+                qdb.exceptions.QiitaDBOperationNotPermittedError):
+            qdb.artifact.Artifact(4).is_submitted_to_ebi
```

Comment:
Completely optional - I personally like to create separate tests for assertRaises and other checks.